piksi_tools/console/settings.yaml

```diff
@@ -2443,3 +2453,98 @@
   default value: N/A
   readonly: true
   Description: inertial navigation system build date
+
+- group: metrics_daemon
+  name: metrics_update_interval
+  expert: true
+  type: integer
+  units: Hz
+  default value: '1'
+  readonly: false
+  Description: Set metric update interval
+  Notes:
+
+- group: metrics_daemon
+  name: enable_log_to_file
+  expert: true
+  type: bool
+  units: N/A
+  default value: 'true'
+  readonly: false
+  Description: Enable metric logging to file
+  Notes:
+
+- group: system
+  name: resource_monitor_update_interval
+  expert: true
+  type: integer
+  units: Hz
+  default value: '0'
+  readonly: false
+  Description: Interval to run the resource monitor at
+  Notes: Value of 0 disables the resource monitor
+
+- group: can0
+  name: mode
+  expert: true
+  type: enum
+  units: N/A
+  default value: 'Disabled'
+  readonly: false
+  enumerated possible values: SBP,NMEA OUT,RTCMv3 IN,RTCMv3 Out
+  Description: Communication protocol for CAN client 0. The client will send packets to a CAN bus.
+  Notes: |
+    "SBP" configures the interface to transmit messages specified in the 'enabled_sbp_messages' setting.
+
+    "NMEA OUT" has no effect for CAN
+
+    "RTCMv3 IN" has no effect for CAN
+
+- group: can0
+  name: enabled_sbp_messages
+  expert: true
+  type: string
+  units: N/A
+  default value: '23,29,65,72,74,81,117,134,136,144,163,165, 166,167,171,175,181,185,187,188,189,190, 257,258,259,520,522,524,526,527,528,1025, 2304,2305,2306,30583,65280,65282,65535' 
```

Comment:
This is probably not what we want. @switanis to comment.

Reply:
I think we only want 72, 74, 117, 522, 527 -- @switanis anything else?